common_features.mk

```
@@ -318,6 +318,11 @@ ifeq ($(strip $(PRINTING_ENABLE)), yes)
     SRC += $(TMK_DIR)/protocol/serial_uart.c
 endif
 
+ifeq ($(strip $(KEY_OVERRIDE_ENABLE)), yes)
+    OPT_DEFS += -DKEY_OVERRIDE_ENABLE
+    SRC += $(QUANTUM_DIR)/key_override.c
```

Based on what this is doing, I feel that moving the files to the `process_keycode` folder would be very appropriate here. 

```suggestion
    SRC += $(QUANTUM_DIR)/process_keycode/key_override.c
```